enhancements/installer/single-node-installation-bootstrap-in-place.md

```diff
@@ -0,0 +1,309 @@
+- Generate ignition - `make generate`
+- Set up networking - `make network` (provides DNS for `Cluster name: test-cluster, Base DNS: redhat.com`)
+- Download rhcos image - `make embed` (download RHCOS liveCD and embed the bootstrap Ignition)
+- Spin up a VM with the the liveCD - `make start-iso`
+- Monitor the progress using `make ssh` and `journalctl -f -u bootkube.service` or `kubectl --kubeconfig ./mydir/auth/kubeconfig get clusterversion`
+
+
+### Risks and Mitigations
+
+*What are the risks of this proposal and how do we mitigate. Think broadly. For
+example, consider both security and how this will impact the larger OKD
+ecosystem.*
+
+*How will security be reviewed and by whom? How will UX be reviewed and by whom?*
+
+## Design Details
+
+### Open Questions
+
+2. Bootable installation artifact?, perhaps the installer should produce the liveCD?
+3. How will the user specify costum configuration, such as installation disk, static IPs? 
+4. Number of revisions for the control plane - do we want to make changes to the bootstrap static pods to make them closer to the final ones?
+6. What do we do with the CRDs that are generated by operators instead of installed from manifests managed by the CVO?
+
+### Test Plan
+
+In order to claim full support for this configuration, we must have
+CI coverage informing the release.
+An end-to-end job using the bootstrap-in-place installation flow,
+based on the [installer UPI CI](https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml#L507),
+ running an appropriate subset of the standard OpenShift tests
+will be created and configured to block accepting release images
+unless it passes.
+
+That end-to-end job should also be run against pull requests for
+the installer and cluster-bootstrap.
+
+### Graduation Criteria
+
+**Note:** *Section not required until targeted at a release.*
+
+Define graduation milestones.
+
+These may be defined in terms of API maturity, or as something else. Initial proposal
+should keep this high-level with a focus on what signals will be looked at to
+determine graduation.
+
+Consider the following in developing the graduation criteria for this
+enhancement:
+
+- Maturity levels
+    - [`alpha`, `beta`, `stable` in upstream Kubernetes][maturity-levels]
+    - `Dev Preview`, `Tech Preview`, `GA` in OpenShift
+- [Deprecation policy][deprecation-policy]
+
+Clearly define what graduation means by either linking to the [API doc definition](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning),
+or by redefining what graduation means.
+
+In general, we try to use the same stages (alpha, beta, GA), regardless how the functionality is accessed.
+
+[maturity-levels]: https://git.k8s.io/community/contributors/devel/sig-architecture/api_changes.md#alpha-beta-and-stable-versions
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
```

Comment:
@eranco74 I will add the following section:
Limitations - where we discuss the fact that some of the CRDs are created by the operators and it will impose a limitation on the user for specifying manifests in advance